web-components/src/components/tooltip/InfoIconTooltip.tsx

```diff
@@ -0,0 +1,19 @@
+import InfoIcon from '../icons/InfoIcon';
+import InfoIconOutlined from '../icons/InfoIconOutlined';
+import InfoTooltip from './InfoTooltip';
+
+interface Props {
+  title: string;
+  outlined?: boolean;
+}
+
+export default function InfoIconTooltip({
```

Comment:
Should we rename this component to `InfoTooltipWithIcon`?
It might be a bit confusing as it is currently, it could be confused with an Icon while it's an actual component.

Reply:
I agree thank you!